SU-SWS/saa_victoria_subtheme
# ADAPT-425 Add mobile styling for Banners

# READY FOR REVIEW 

# Summary
- Add mobile styling for Banner C Left and testimonial banner

# Review By (Date)
- 6/30

# Urgency
- low

# Steps to Test
1. Pull branch to local
2. View homepage banner (check for mobile)
3. Create/edit a page and add a Paragraph banner. 
4. View testimonial banner (check for mobile)

# Associated Issues and/or People
- [ADAPT-425](https://stanfordits.atlassian.net/browse/ADAPT-425)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
